src/app/screens/Bloc/index.js

```diff
@@ -4,12 +4,19 @@ import i18 from 'i18next';
 
 import InlineInput from 'app/components/InlineInput';
 import InlineTextArea from 'app/components/InlineTextArea';
+import NotesActions from 'redux/notes/actions';
+import { handleTextStyle, countWords } from 'utils/functionUtils';
 
 import styles from './styles.module.scss';
 
-const Bloc = () => {
+const Bloc = ({ dispatch }) => {
   const [titleValue, setTitleValue] = useState('');
   const [textValue, setTextValue] = useState('');
+  const [textClassNames, setTextClassNames] = useState([]);
+
+  const setTextStyle = textStyle => {
+    setTextClassNames(handleTextStyle(textStyle, textClassNames));
+  };
```

Comment:
Remember that you should use () instead of {} when you return the value directly